lips/lip-8.md

```diff
@@ -205,8 +282,16 @@ The [PaymentActionObject](https://lip.libra.org/lip-1/#paymentactionobject) now
 
 The auth/capture flow now adds the following to the status enum of [PaymentActor](https://lip.libra.org/lip-1/#paymentactorobject):
 
-* `authorized` - Payment amount is authorized but not yet captured.
+* `authorized` - The payment amount is authorized but not yet captured.
 
 `abort` may still be used to cancel the authorization early.  Once a capture action occurs, the status of the payment will now be updated to `ready_for_settlement`.
 
-**Valid Status Transitions**. `authorized` is now a valid initial value and may be followed by `ready_for_settlement` (upon a successful capture) or `abort` (if one side wishes to cancel the auth).
+**Valid Status Transitions**. `authorized` is now a valid initial value and may be followed by `ready_for_settlement` (upon a successful capture) or `abort` (if a valid cancel request was sent).
```

Comment:
after reading the whole auth/capture section, I still do not understand how the valid status transition. Can we have a detailed diagraph like we are making for lip-1?